doc/source/api/general/case_reader.rst

```diff
@@ -17,8 +17,8 @@ from the ``examples`` repository:
   >>> from ansys.fluent.core import examples
   >>> from ansys.fluent.core.filereader.case_file import CaseFile
 
-  >>> case_filepath = examples.download_file("Static_Mixer_Parameters.cas.h5", "pyfluent/static_mixer")
-  >>> reader = CaseFile(case_filepath=case_filepath)
+  >>> case_file_path = examples.download_file("Static_Mixer_Parameters.cas.h5", "pyfluent/static_mixer")
```

Comment:
@hpohekar If you read the issue carefully, you'll see that it's requesting that we use the term name and not path. This is consistent with the settings API. So this would be `case_file_name`.

Reply:
@seanpearsonuk `examples.download_file` returns `file_path` unless `return_only_file_name` argument is specified as 'True`.

![image](https://github.com/ansys/pyfluent/assets/106588300/badf9d5a-b99a-4380-9151-7ca720d95f3c)

To get case_file_name we should pass `return_only_file_name` as `True`.

Reply:
Same is the case with `import_file_name`, wherever we have used `import_file_name`, it is actually file path not the file name. It is inconsistent with documentation of `examples.py` (shared in above screenshot).

Reply:
Following screenshot demonstrates correct use of `case_file_name`

![image](https://github.com/ansys/pyfluent/assets/106588300/0f6708b2-0140-47a9-9cff-c8c13c49f78a)

Reply:
@hpohekar Please correct me if I'm wrong, but I believe that in `solver.file.read_case(file_name=file_name)`, `file_name` can be a path. The issue I created intended that we be consistent with that.

Reply:
@seanpearsonuk Right. I will update this change. Thank you.

Reply:
Done. Thank you.

Reply:
You're right @hpohekar: `download_file` was previously using different names for "file name+path" = `filepath`, versus "only file name" = `filename`.

To perhaps make things a bit clearer/less confusing here, it might be better to rename `return_only_file_name` to `return_without_path`. 

Therefore, when `return_without_path` is unspecified or `False`, then `file_name` will include the full path, and when `return_without_path` is `True`, `file_name` will not include the full path.

Reply:
@raph-luc Done. Thank you.